include/setting_option.h

```diff
@@ -9,17 +9,17 @@ namespace myrps
 {
 enum class SettingOption
 {
-  D_EASY,
-  D_MEDIUM,
-  D_HARD,
+  D_RANDOM,
```

Comment:
Please use `camelCased` with 'k' preface for enum naming, [per the style guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names), and [precedence set in previously named enums](https://github.com/drofp/MyRPS/blob/master/include/move.h)

Reply:
done